src/components/product-list/ProductListItem.tsx

```diff
@@ -0,0 +1,32 @@
+import { Product } from '@common/type';
+import { Box, Card } from '@mui/material';
+import { cardContainer, productInfoContainer } from '@styles/all-products/productListItem';
+import CardImage from './CardImage';
+import ProductDescription from './ProductDescription';
+import ProductPriceInfo from './ProductPriceInfo';
+
+interface ProductListItemProps {
+  product: Product;
+}
+
+export function ProductListItem(props: ProductListItemProps) {
+  const { product } = props;
+
+  const isPcsAvailable = () => {
+    if (product.price.pcs && product.stock.pcs) {
+      return true;
+    } else if (product.price.kg && product.stock.kg) {
+      return false;
+    }
+  };
```

Comment:
const isPcsAvailable  = product.price.pcs && product.stock.pcs || !product.price.kg && product.stock.kg

Reply:
okay, thank you
I need to learn more about this